govcd/system.go

```diff
@@ -8,11 +8,26 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"regexp"
 	"strings"
 
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
 )
 
+// Simple structure to pass Edge Gateway creation parameters.
+type EdgeGatewayCreation struct {
+	ExternalNetworks          []string
+	OrgName                   string
+	VdcName                   string
+	Name                      string
+	Description               string
+	BackingConfiguration      string
+	AdvancedNetworkingEnabled bool
+}
+
+// List of allowed values for GatewayBackingConfig.
+var EGWAllowedBackingConfig = []string{"compact", "full"}
```

Comment:
Did you figure out how to match other in UI visible configs?

Reply:
As mentioned above, these are the only values accepted. 
The values in the GUI refer to something else.